test/e2e/specs/wp-calypso-gutenberg-checkout-spec.js

```diff
@@ -39,7 +39,7 @@ describe( `[${ host }] Calypso Gutenberg Editor: Checkout on (${ screenSize }) i
 	this.timeout( mochaTimeOut );
 	let editorUrl;
 
-	describe( 'Can trigger the checkout modal via post editor', function () {
+	describe.skip( 'Can trigger the checkout modal via post editor', function () {
```

Comment:
Are we not better skipping the parent `describe` so subsequent tests are not run here as they will all fail if we just skip this one?

Reply:
Good call, fixed it 👍